content/guides/userguides/accessing-metadata-via-tdr.md

```diff
@@ -0,0 +1,127 @@
+---
+date: "2021-12-25"
+description: "Overview of accessnig proejct metadata via the Terra Data Repository (TDR)."
+draft: false
+title: "Accessing Metadata via TDR"
+---
+
+# Accessing Metadata via TDR
+
+The [Terra Data Repository](https://support.terra.bio/hc/en-us/sections/4407099323675-Terra-Data-Repository)
+(TDR) offers a complete and authoritative source for the metadata made 
+available by the Data Portal data browser. This guide will walk you through the 
+process of accessing metadata stored within TDR.
+
+For detailed information regarding the structure of the data within TDR, see the
+[Terra Data Repository schema](https://github.com/HumanCellAtlas/dcp2/blob/main/docs/dcp2_system_design.rst#terra-data-repository-schema).
+
+
+## Running BigQuery SQL Queries
+
+Access to the metadata in TDR can be accomplished using [Google BigQuery](https://cloud.google.com/bigquery/docs/introduction).
+Queries can be run via the Cloud Console or by using the command-line tool `bq`.
+- [Using the Cloud Console](https://cloud.google.com/bigquery/docs/bigquery-web-ui)
+- [Using the bq command-line tool](https://cloud.google.com/bigquery/docs/bq-command-line-tool)
+
+
+## Finding a Catalog's Source
+
+Each project in the data browser belongs to a catalog. Each catalog consists of
+one or more sources, and each source is composed of properties. These properties
+include the name of the Google Cloud project and the name of the TDR snapshot.
+
+The source of a catalog can be obtained via the [Data Browser API](https://data.humancellatlas.org/apis/api-documentation/data-browser-api) 
+`/index/catalogs` endpoint. 
+
+Using `curl` to make the request, and `jq` to parse the response, the first step
+is to identify the current default catalog.
+```
+$ curl -s https://service.azul.data.humancellatlas.org/index/catalogs | jq '.default_catalog'
+"dcp11"
+```
+
+With the name of the catalog we can now parse out the catalog's source(s) from
+the endpoint response.
+
+```
+$ curl -s https://service.azul.data.humancellatlas.org/index/catalogs | jq '.catalogs.dcp11.plugins.repository.sources'
+[
+  "tdr:tdr-fp-c315dee1:snapshot/hca_prod_20201120_dcp2___20211101_dcp11:/2"
+]
+```
+
+In this example, there is one source for the `dcp11` catalog. The name of the
+project (`tdr-fp-c315dee1`) and name of the snapshot (`hca_prod_20201120_dcp2___20211101_dcp11`)
+can be extracted from the source following the syntax below.
+
+```
+"tdr:{PROJECT_NAME}:snapshot/{SNAPSHOT_NAME}:{prefix}"
+```
+
+Alternatively, the project and snapshot details can be obtained via the
+[TDR API](https://jade-terra.datarepo-prod.broadinstitute.org/swagger-ui.html#/snapshots/enumerateSnapshots)
+`enumerate snapshots` endpoint. 
```

Comment:
I'm not sure how useful/appropriate it is to include this link here. In order to search TDR for the relevant sources, the user would have to already know its name and/or ID, so I don't think it makes sense to present this as an alternative way to obtain that information. Perhaps just change the context so that this is presented as a way for users to learn more detailed information about the underlying snapshot if they want more info than is included in our API response.